From ff68717871789b6c3b1b3d66ff399621042c7b61 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Wed, 21 Jun 2023 19:02:19 +0200
Subject: [PATCH 0405/1141] arm64: dts: st: swap lvds & dsi on stm32mp257f-ev1
 board

To get the HDMI bridge on first display node, dsi & lvds endpoints
are permuted.

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Change-Id: I0bb510c07317a58e03b9c5b362cf4bfaa9ca1e7e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/314676
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 940a64e78438..9b07cd272ebb 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -182,7 +182,7 @@ ports {
 		port@0 {
 			reg = <0>;
 			dsi_in: endpoint {
-				remote-endpoint = <&ltdc_ep1_out>;
+				remote-endpoint = <&ltdc_ep0_out>;
 			};
 		};
 
@@ -379,12 +379,12 @@ port {
 
 		ltdc_ep0_out: endpoint@0 {
 			reg = <0>;
-			remote-endpoint = <&lvds_in>;
+			remote-endpoint = <&dsi_in>;
 		};
 
 		ltdc_ep1_out: endpoint@1 {
 			reg = <1>;
-			remote-endpoint = <&dsi_in>;
+			remote-endpoint = <&lvds_in>;
 		};
 	};
 };
@@ -399,7 +399,7 @@ ports {
 		port@0 {
 			reg = <0>;
 			lvds_in: endpoint {
-				remote-endpoint = <&ltdc_ep0_out>;
+				remote-endpoint = <&ltdc_ep1_out>;
 			};
 		};
 
-- 
2.39.2

